File: 7KYU/Simple Fun #10: Range Bit Counting/solution.cs
using System.Linq;
namespace myjinxin
{
    using System;
    
    public class Kata
    {
        public int RangeBitCount(int a,int b)
          => Enumerable.Range(a, (b-a)+1)
                       .Select(c => Convert.ToString(c, 2)
                                           .Replace("0", "")
                                           .Length)
                       .Sum();
    }
}

File: 7KYU/Simple Fun #10: Range Bit Counting/solution.go
package kata

import "strconv"

func countOnes(s string) int {
	sum := 0
	for _, c := range s {
		if c == '1' {
			sum += 1
		}
	}
	return sum
}

func RangeBitCount(a,b int) int {
  noOnes := 0 
  for i:=a; i <=b; i++ {
    bin := strconv.FormatInt(int64(i), 2)
    noOnes += countOnes(bin)    
  }
  return noOnes
}